express/code/blocks/page-list/page-list.js

```diff
@@ -19,7 +19,10 @@ async function fetchIndex(indexURL) {
 function outputPages(filteredPages, $block) {
   filteredPages.forEach((page) => {
     const p = createTag('p');
-    p.innerHTML = `<a href="${page.path}">${page.shortTitle}</a>`;
+    const a = createTag('a');
```

Comment:
not tested, but you simplify to something like 
createTag('a', { href: path.path }, page.shortTitle);